src/QsCompiler/Tests.LanguageServer/ProjectLoaderTests.cs

```diff
@@ -102,10 +102,6 @@ public void SupportedTargetFrameworks()
             var loader = new ProjectLoader();
             Assert.IsTrue(loader.IsSupportedQsFramework("netstandard2.0"));
             Assert.IsTrue(loader.IsSupportedQsFramework("netstandard2.1"));
-            Assert.IsTrue(loader.IsSupportedQsFramework("netcoreapp2.0"));
-            Assert.IsTrue(loader.IsSupportedQsFramework("netcoreapp2.1"));
-            Assert.IsTrue(loader.IsSupportedQsFramework("netcoreapp2.2"));
-            Assert.IsTrue(loader.IsSupportedQsFramework("netcoreapp3.0"));
```

Comment:
Are we dropping support for the .net core 2.0, 2.1, 2.2, and 3.0? What is the impact / significance of removing these lines from the project loader tests?